tls/s2n_kem_preferences.c

```diff
@@ -39,6 +39,21 @@ const struct s2n_kem *pq_kems_r2r1_2020_07[5] = {
     &s2n_sike_p503_r1,
 };
 
+const struct s2n_kem *pq_kems_r3r2r1_2021_05[7] = {
+    /* Round 3 Algorithms */
+    &s2n_kyber_512_r3,
+    &s2n_bike_l1_r3,
+
+    /* Round 2 Algorithms */
+    &s2n_kyber_512_r2,
+    &s2n_bike1_l1_r2,
+    &s2n_sike_p434_r3,
```

Comment:
Should `s2n_sike_p434_r3` be up higher with the other round 3 KEMs?

Reply:
The problem is older Round 2 clients that announce support for Kyber_R2, BIKE_R2, and SIKE_R2 (which is equivalent to SIKE_R3 and uses the same algorithm ID). If we move `s2n_sike_p434_r3` above `s2n_kyber_512_r2`, then these older clients who only support Round 2 KEM's, and who haven't changed anything, will stop negotiating Kyber_R2 and will start negotiating SIKE_R2 and see a large performance drop.